modules/op-release-notes-1-8.adoc

```diff
@@ -50,8 +254,146 @@ Workaround: Use `nodejs:14-ubi8` instead of the `nodejs:14-ubi8-minimal` image s
 // (link:https://issues.redhat.com/browse/SRVKP-1782[#SRVKP-1782])
 // Writer, also see https://github.com/tektoncd/operator/blob/main/cmd/openshift/operator/kodata/tekton-addon/addons/02-clustertasks/source_local/s2i-nodejs/s2i-nodejs-task.yaml. Overkill: The user can run the `$ oc edit clustertask s2i-nodejs` command and replace `$(params.VERSION)` with `14-ubi8`. Alternatively, you can use a `TaskRun` resource to set the tag of the `nodejs` image stream to `14-ubi8`.
 
-// [id="fixed-issues-1-8_{context}"]
-// == Fixed issues
+// .Operator
+* The `s2i-nodejs` image cannot use the `nodejs:14-ubi8-minimal` image stream to perform S2I builds. Doing so produces an `error building at STEP "RUN /usr/libexec/s2i/assemble": exit status 127` message. There is no workaround at this time.
+// borrowed from https://issues.redhat.com/browse/SRVKP-1782
+// Pavol Pitoňák
+
+[id="fixed-issues-1-8_{context}"]
+== Fixed issues
+
+// .Pipelines
+
+* Before this update, the metrics for pipeline runs in the Developer view of the web console were incomplete and outdated. With this update, the issue has been fixed so that the metrics are correct.
+// link:https://issues.redhat.com/browse/SRVKP-1789
+//  Khurram Baig
+
+* Before this update, if a pipeline had two parallel tasks that failed and one of them had `retries=2`, the final tasks never ran, and the Pipeline failed from a timeout. For example, the `pipelines-operator-subscription` task failed intermittently with an "Unable to connect to the server: EOF" error. With this update, the issue has been fixed so that the final tasks always run.
+// link:https://issues.redhat.com/browse/SRVKP-1724
+// Vincent Demeester
+
+* Before this update, when a pipeline run was in a "stopping" state, that is, if a task run failed, other task runs might not complete their retries. As a result, no `finally` tasks were scheduled, which caused the pipeline to hang. This update resolves the issue. `TaskRuns` and `Run` objects can retry when a pipeline run has stopped, even by  graceful stopping, so that pipeline runs can complete.
```

Comment:
```suggestion
* Before this update, when a pipeline run was in a "stopping" state, that is, if a task run failed, other task runs might not complete their retries. As a result, no `finally` tasks were scheduled, which caused the pipeline to hang. This update resolves the issue. `TaskRuns` and `Run` objects can retry when a pipeline run has stopped, even by graceful stopping, so that pipeline runs can complete.
```

Reply:
Updated. Thanks.